geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/parameters/ClusterParameterRequirements.java

```diff
@@ -27,7 +27,7 @@ public void checkParameters(Command command, ExecutionHandlerContext context) {
 
     if (numberOfArguments < 2) {
       throw new RedisParametersMismatchException(command.wrongNumberOfArgumentsErrorMessage());
-    } else if (numberOfArguments > 2) {
+    } else if (numberOfArguments > 3) {
```

Comment:
It is not clear to me why this change was made. If it was wrong before should a test have been broken? Can we add a test to the integration test that passes CLUSTER the wrong number of parameters?

Reply:
I've added some more tests to the `AbstractClusterIntegrationTest`. I changed this `ClusterParameterRequirements` class because the command `CLUSTER KEYSLOT XXX` has three arguments. I added additional validation into the `ClusterExecutor`